pkg/validation/internal/testdata/badName.csv.yaml

```diff
@@ -0,0 +1,292 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: someoperatorwithanextremelylongnamethatmakenosensewhatsoever.v999.999.999
```

Comment:
Shouldn't this file just contain the necessary fields, ex. just GVK and `metadata.name`?

Reply:
You want me to slim this down to just basic stuff. Yah, I can do that.

Reply:
I stripped it down as much as I could. This CSV still goes through the typical validation for the whole CSV so a few things still need to be there.